cmd/mattermost-mattermod/main.go

```diff
@@ -71,6 +72,15 @@ func main() {
 		}
 	}()
 
+	c := cron.New()
+
+	_, err = c.AddFunc("10 2 * * *", s.RefreshMembers)
+	if err != nil {
+		mlog.Error("failed adding RefreshMembers cron", mlog.Err(err))
+	}
+
+	c.Start()
+
 	sig := make(chan os.Signal, 1)
 	signal.Notify(sig, os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
 
```

Comment:
For completeness, should we `c.Stop()` after shutting down?

Reply:
c.Stop() is not called also for jobserver. I will create a followup ticket to call Stop after shutdown.